JavaScript/package.json

```diff
@@ -76,24 +88,18 @@
 		"tslint-config-prettier": "^1.4.0",
 		"tslint-config-standard": "^6.0.0",
```

Comment:
Please, check if you still need the `tslint-*` dependencies

Reply:
thx @enzocanoo we will review that